examples/sample-pub/sample-pub-app.js

```diff
@@ -46,6 +41,14 @@ const SWG_JS_URLS = {
   tt: 'https://news.google.com/swg/js/v1/swg-tt.js',
 };
 
+const SWG_GSI_JS_URLS = {
+  local: '/dist/subscriptions-gsi.max.js',
+  local_min: '/dist/subscriptions-gsi.js',
+  prod: 'https://news.google.com/swg/js/v1/swg-gsi.js',
+  autopush: 'https://news.google.com/swg/js/v1/swg-gsi-autopush.js',
```

Comment:
Do publishers need these? Wouldn't they only use the prod version and not even the local version? The shouldn't be messing with anything in the helper file.

Reply:
SwG developers need these